modules/nw-ovn-kubernetes-alerts-cli.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-troubleshooting-sources.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ovn-kubernetes-alerts-cli_{context}"]
+= Viewing OVN-Kubernetes alerts in the CLI
+
+You can get information about alerts and their governing alerting rules and silences from the command line.
+
+.Prerequisites
+
+* Access to the cluster as a user with the cluster-admin role.
+* The OpenShift CLI (`oc`) installed.
+* You have installed `jq`.
+
+.Procedure
+
+. View active or firing alerts by running the following commands.
+
+.. Set the alert manager route environment variable by running the following command:
++
+[source,terminal]
+----
+$ ALERT_MANAGER=$(oc get route alertmanager-main -n openshift-monitoring \
+-o jsonpath='{@.spec.host}')
+----
+
+.. Issue a `curl` request to the alert manager route API with the correct authorization details requesting specific fields by running the following command:
++
+[source,terminal]
+----
+$ curl -s -k -H "Authorization: Bearer \
```

Comment:
Error: must specify one of -f and -k
error: unknown command "token prometheus-k8s"
See 'oc create -h' for help and examples
parse error: Invalid numeric literal at line 2, column 10